core/embed/boardloader/memory.ld

```diff
@@ -3,9 +3,10 @@
 ENTRY(reset_handler)
 
 MEMORY {
-  FLASH  (rx)  : ORIGIN = 0x08000000, LENGTH = 48K
-  CCMRAM (wal) : ORIGIN = 0x10000000, LENGTH = 64K
-  SRAM   (wal) : ORIGIN = 0x20000000, LENGTH = 192K
+  FLASH  (rx)     : ORIGIN = 0x08000000, LENGTH = 48K
+  CCMRAM (wal)    : ORIGIN = 0x10000000, LENGTH = 64K - 0x100
+  BOOT_ARGS (wal) : ORIGIN = 0x1000FF00, LENGTH = 0x100
```

Comment:
i don't think we want this for f4 boardloader

Reply:
I've left it intentionally here, but yes, it's not necessary and maybe confusing. Removed  66381328fab284b792145d8f3d733fb036cb4921

Reply:
that commit removes it from bootloader, not boardloader. 

(i don't want it in boardloader because it again prevents the end of ccmram to be cleared)

Reply:
Yeah, I was too fast. fixed d52365f2dc54c582266b47b9f154c6e893684663